core/themes/basis/template.php

```diff
@@ -53,6 +66,13 @@ function basis_preprocess_page(&$variables) {
   }
 }
 
+/**
+ * Every time we add supplemental CSS we add the version number here.
+ */
+function basis_supplemental_css_versions() {
+  return array('No updates', '1.23', '1.28.4', '1.30','1.31.5');
```

Comment:
I feel like these numbers should be reversed. The current UI:

![image](https://github.com/user-attachments/assets/856db44e-7e33-4c80-828a-b1a0b8dadc07)

Out of the box, the newest version will be the default option, so it should be the first item in the list rather than the last. As a user, I think it's more likely that people will more regularly roll back one or two releases when they need to fix regressions, so putting those options nearer the top makes sense to me.